model/project_event.go

```diff
@@ -22,14 +22,87 @@ type ProjectSettings struct {
 	Subscriptions      []event.Subscription `bson:"subscriptions" json:"subscriptions"`
 }
 
+type ProjectSettingsEvent struct {
+	ProjectSettings              `bson:",inline"`
+	FilesIgnoredFromCacheDefault bool `bson:"files_ignored_from_cache_default,omitempty" json:"files_ignored_from_cache_default,omitempty"`
```

Comment:
re chaya's comment thought, I think there should also be a comment here to explain the bools